CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ CHANGELOG
 =========
 
 * dev-develop
+    * FEATURE #68 [General]   Introduce decorators for index names and introduce prefixes
```

Comment:
I doubt that the content in square brackets is useful here, because the components are not as clearly separated as in Sulu. I would omit them, WDYT @chirimoya @wachterjohannes